src/unity.yml

```diff
@@ -112,12 +116,12 @@ components:
       $ref: './unity/schemas/Invoice.yml'
     Invoices:
       $ref: './unity/schemas/Invoices.yml'
-    User:
-      $ref: './unity/schemas/User.yml'
+    OrgUser:
```

Comment:
the response from the users endpoints on the users page returns what we refer to as OrgUsers vs the operator user schemas which reflect quartz users. There are some differences with those schemas so they were separated.